testconfig.yml

```diff
@@ -11,97 +11,129 @@ runs:
         duration:
           description: Time (s)
           chart: boxplot
+        alloc:
+          description: Allocated memory (bytes)
+          chart: boxplot
     BenchmarkBuildMultistage:
       description: Multistage build
       count: 4
       metrics:
         duration:
           description: Time (s)
           chart: boxplot
+        alloc:
+          description: Allocated memory (bytes)
+          chart: boxplot
```

Comment:
Atm it will create a chart for each metric:

![image](https://github.com/user-attachments/assets/8efbdf1d-cdc4-479b-837a-e849baf1b272)

Which is not optimal to link different metrics within the same run. I'm looking to create overlays as follow-up.